nexus/tests/config.test.toml

```diff
@@ -46,3 +46,7 @@ level = "trace"
 mode = "file"
 if_exists = "fail"
 path = "UNUSED"
+
+# Configuration for interacting with the timeseries database
+[timeseries_db]
+address = "[::1]:8123"
```

Comment:
Does this use a scheme similar to the dropshot servers, where we bind to 0, figure out the port, and update the config to reflect which port it got?

If not, how does this work with concurrent tests that both start up Clickhouse?

Reply:
Oh good catch, I did the same addition to every config.toml. I think this works for concurrent tests because the nexus internal `oximeter_db::Client` is never used in those tests. The `ClickHouseInstance` that the test context creates does get a random port from the OS, which is used in the existing tests. But there's no test which verifies that _nexus_ can reach the DB and run queries. I'll add one.

Reply:
I've updated this to port 0 and then collect the actual port in Nexus tests, similar to how it's done for CRDB. That's in 0a4f203. I'd like to add a test for this nexus method, but I'm not sure how to do that yet. We could test the Dropshot endpoint, but that requires either using the `nexus-client` crate or something like `reqwest`. The former doesn't work with the dependency graph we're trying to keep. I'll sketch out a test with the latter, and see how it works.

Reply:
Forgot about the `ClientTestContext`. Added an integration test in 8d3bb4e.